### ISSUE TITLE
Search doesn't work at Chinese character

### ISSUE THREAD

**Issue by Elcvise** [Ranchero-Software/NetNewsWire#4080]
Tap the top bar for Article Search, and English and Numbers works great, but when I type the Chinese key words,it doesn’t work.

**Comment by suandqf** [comment]
这问题都好多年了，不过除了这其它倒还好。

**Comment by sunyunxian** [comment]
hope develop this feature.

**Comment by LinxHex** [comment]
Yes — the Chinese text search indeed has issues, but the macOS version is working fine, and the iOS version is the one having problems.
